blocks/shopify/src/style.scss

```diff
@@ -0,0 +1,49 @@
+/**
+ * The following styles get applied both on the front of your site
+ * and in the editor.
+ *
+ * Replace them with your own styles or remove the file completely.
+ */
+
+// .wp-block-cata-shopify {}
```

Comment:
There are some commented out class names here and then there are styles for them later on.

Reply:
Thanks, I must have overlooked this after I copied from the products block